src/js/TuningSliders.js

```diff
@@ -62,12 +62,21 @@ TuningSliders.initialize = function() {
     this.updateFilterSlidersDisplay();
 };
 
-TuningSliders.setExpertMode = function(expertMode) {
-    this.expertMode = expertMode;
-    if (this.expertMode) {
-        $('#slidersPidsBox, #slidersFilterBox').removeClass('nonExpertModeSliders');
+TuningSliders.setExpertMode = function() {
+    this.expertMode = isExpertModeEnabled();
+    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+        const dMinShow = parseInt($('.pid_tuning input[name="dMinRoll"]').val()) !== 0;
+        $('.tab-pid_tuning .MasterSlider').toggle(this.expertMode);
+        $('.tab-pid_tuning .DMinRatioSlider').toggle(this.expertMode && dMinShow);
+        $('.tab-pid_tuning .advancedSlider').toggle(this.expertMode);
     } else {
-        $('#slidersPidsBox, #slidersFilterBox').addClass('nonExpertModeSliders');
+        if (this.expertMode) {
+            $('#slidersPidsBox, #slidersFilterBox').removeClass('nonExpertModeSliders');
+        } else {
+            $('#slidersPidsBox, #slidersFilterBox').addClass('nonExpertModeSliders');
+        }
```

Comment:
For this kind of if-else, maybe is easier too use a toggle too:

```suggestion
        $('#slidersPidsBox, #slidersFilterBox').toggleClass('nonExpertModeSliders', this.expertMode);
```

All the parts when we can use a `toggle` and not an if reduces the complexity of the code.

Reply:
done